src/fs_post.glsl

```diff
@@ -0,0 +1,23 @@
+in vec2 v_uv;
+
+uniform sampler2D tex_col;
+uniform vec2 pixel_size;
+
+out vec4 frag_color;
+void main() {
+    float x = pixel_size.x * 0.25;
+    float y = pixel_size.y * 0.25;
+    frag_color = vec4(
+            4.0 * texture(tex_col, v_uv).rgb +
+
+            2.0 * texture(tex_col, v_uv + vec2( x,  0)).rgb +
+            2.0 * texture(tex_col, v_uv + vec2(-x,  0)).rgb +
+            2.0 * texture(tex_col, v_uv + vec2( 0, -y)).rgb +
+            2.0 * texture(tex_col, v_uv + vec2( 0,  y)).rgb +
+
+            1.0 * texture(tex_col, v_uv + vec2( x,  y)).rgb +
+            1.0 * texture(tex_col, v_uv + vec2(-x,  y)).rgb +
+            1.0 * texture(tex_col, v_uv + vec2(-x, -y)).rgb +
+            1.0 * texture(tex_col, v_uv + vec2( x, -y)).rgb
+    , 16.0) / 16.0;
```

Comment:
Maybe this should be an identity shader? Or is it a good test?

Reply:
I wonder if we'll get back to this. I think we should remove it from this PR and allow an engine user to use their own shader.

Reply:
https://github.com/sornas/lingon/tree/post-shaders contains the shader code, for future reference.